sdk/core/azure-core/azure/core/tracing/common.py

```diff
@@ -0,0 +1,79 @@
+# --------------------------------------------------------------------------
```

Comment:
```py
# ------------------------------------
# Copyright (c) Microsoft Corporation.
# Licensed under the MIT License.
# ------------------------------------
```

Reply:
done

Reply:
oops yes. So I was adding the headers and I copied it from `azure.core.settings`. Now I do my headers inconsistently. If you want I can go back and change all of my files to have the smaller header. The point is that azure core files all have different copyright header.